helper/schema/valuetype_string.go

```diff
@@ -0,0 +1,16 @@
+// generated by stringer -type=ValueType; DO NOT EDIT
+
+package schema
+
+import "fmt"
+
+const _ValueType_name = "TypeInvalidTypeBoolTypeIntTypeStringTypeListTypeMapTypeSettypeObject"
```

Comment:
Out of curiosity, is the last `type` in this correct, being lowercased rather than uppercased like the rest?